app/components/forms/wizard/basic-details-step.js

```diff
@@ -121,6 +121,15 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   },
 
   getValidationRules() {
+    window.$.fn.form.settings.rules.checkMaxMin = () => {
+      let { tickets } = this.tickets;
```

Comment:
Incorrect. 

Should be 

```js
let { tickets } = this;
```

But then, why do you even need to store it in a variable ? You can directly use `this.tickets` everywhere right ?

Reply:
@niranjan94 Yes, you're right.